bigtable-hbase-parent/bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/RowMergerPerf.java

```diff
@@ -21,26 +23,37 @@
 public class RowMergerPerf {
 
   public static void main(String[] args) {
-    for (int i = 0; i < 10; i++) {
-      rowMergerPerf(createResponses());
+    for(int i =0; i<5; i++) {
+      rowMergerPerf(createResponses(1));
+    }
+    for (int i = 1; i <= 101; i += 10) {
+      System.out.println("===================");
+      System.out.println("using " + i + " Cells");
+      rowMergerPerf(createResponses(i));
     }
   }
 
-  private static List<ReadRowsResponse> createResponses() {
+  private static List<ReadRowsResponse> createResponses(int cellCount) {
+    Builder readRowsResponse = ReadRowsResponse.newBuilder();
 
-    CellChunk contentChunk = CellChunk.newBuilder()
-        .setFamilyName(StringValue.newBuilder().setValue("Family1"))
-        .setQualifier(BytesValue.newBuilder().setValue(ByteString.copyFromUtf8("Qaulifier")))
-        .setRowKey(ByteString.copyFrom(Bytes.toBytes("rowkey-0")))
-        .setValue(ByteString.copyFrom(RandomStringUtils.randomAlphanumeric(10000).getBytes()))
-        .setTimestampMicros(0L)
-        .setCommitRow(true)
-        .build();
+    int size = 100_000 / cellCount;
```

Comment:
I added a Precondition for cellCount > 0.  I added a comment that 100_000 % cellCount !=0 is fine.